### PR TITLE
Fix FieldMask issues

### PR DESCRIPTION
**Pull request by dmdashenkov**
Fixed `FieldMasks` coverage: 
 - Test for selecting single message from `Stand`.
 - Test applying an empty `FieldMask` to a collection of messages.

Added exception logging for `FieldMasks`.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol PTAL

File: server/src/test/java/org/spine3/server/stand/StandShould.java [hunk 1]
```diff
@@ -561,7 +562,47 @@ public void onNext(QueryResponse value) {
...
-        requestSampleCustomer(new int[] {}, getDuplicateCostumerStreamObserver());
+        requestSampleCustomer(new int[]{}, getDuplicateCostumerStreamObserver());
+    }
+
+    @SuppressWarnings("MethodWithMultipleLoops")
+    @Test
+    public void select_entity_singleton_by_id_and_apply_field_masks() {
+        final Stand stand = prepareStandWithAggregateRepo(InMemoryStandStorage.newBuilder()
+                                                                              .build());
+        final String customerFqn = Customer.getDescriptor()
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/stand/StandShould.java]: Why is `Fqn` and what is he doing here?

File: server/src/test/java/org/spine3/server/stand/StandShould.java [hunk 2]
```diff
@@ -561,7 +562,47 @@ public void onNext(QueryResponse value) {
...
+        final String[] paths = {customerFqn + ".id", customerFqn + ".name"};
+        final FieldMask fieldMask = FieldMask.newBuilder()
+                                             .addAllPaths(Arrays.asList(paths))
+                                             .build();
+
+        final List<Customer> customers = new LinkedList<>();
+        final int count = 10;
+
+        for (int i = 0; i < count; i++) {
+            // Has new id each time
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/stand/StandShould.java]: `ID`.

File: server/src/main/java/org/spine3/server/entity/FieldMasks.java [hunk 3]
```diff
@@ -160,15 +164,37 @@ private FieldMasks() {
...
     }
+
+    private static Logger log() {
+        return LogSingleton.INSTANCE.value;
+    }
+
+    private enum LogSingleton {
+        INSTANCE;
+        @SuppressWarnings("NonSerializableFieldInSerializableClass")
+        private final Logger value = LoggerFactory.getLogger(SubscriptionService.class);
```
Comment (armiol) [inline server/src/main/java/org/spine3/server/entity/FieldMasks.java]: `SubscriptionService`? Really?

File: server/src/main/java/org/spine3/server/entity/FieldMasks.java [hunk 4]
```diff
@@ -160,15 +164,37 @@ private FieldMasks() {
...
-        } catch (ClassNotFoundException | ClassCastException e) {
+        } catch (ClassNotFoundException e) {
+            final String message = String.format(
+                    "Class for name %s could not be found. Try to rebuild the project. Make sure \"known_types.properties\" exists.",
+                    className);
+            log().warn(message, e);
+            builderClass = null;
+        } catch (ClassCastException e) {
+            final String message = String.format(
+                    "Class %s Must be assignable from com.google.protobuf.Message. Try to rebuild the project. Make sure type URL is valid.",
```
Comment (armiol) [inline server/src/main/java/org/spine3/server/entity/FieldMasks.java]: Please extract string literals as constants.

**Comment by dmdashenkov** [comment]
@armiol PTAL again

File: server/src/test/java/org/spine3/server/stand/StandShould.java (hunk 1 above)
Comment (armiol) [inline server/src/test/java/org/spine3/server/stand/StandShould.java]: Could you please either rename it to `customerDescriptor` or add a line comment such as `//Fully qualified name for {@code Customer} message`.

**Comment by armiol** [comment]
@dmdashenkov looks good except for the variable naming. Please see my comment.

**Pull request merged**